lib/utilities/walk-up-path.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var path = require('path');
+
+function walkUp(thePath) {
```

Comment:
> Doesn't address leading slashes, only relative paths.

Do we want to worry about that complexity if it's never being called that way? Are you thinking of the future?